Specs/folly/spectrum-folly.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |spec|
   spec.name = "spectrum-folly"
-  spec.version = "2018.11.12.00"
+  spec.version = "2019.01.21.00"
```

Comment:
Updating the specs here will require a `pod trunk push` to update the global cocoa pods trunk and then a modification of `SpectrumCore.podspec` & `SpectrumKit.podspec` to update the pinned version. Considering we're the only ones being able to update the `spectrum-folly` spec it might unfortunately be easier if we just do this ourselves before the next release (at least for iOS).

Reply:
Missed those for some reason, added them. Can't CocoaPods be updated/added automatically on release? Current system sounds like a nightmare to maintain.

Reply:
We can certainly automate it, we just haven't done so yet.

Reply:
Eventually we'd like to move off having other dependencies specs defined in Spectrum's repo.

Reply:
@cuva Great! Where can I find the new checksum (or can you add it yourself)?

Reply:
Running `pod install` in `ios/SpectrumKitSample` should update the checksums accordingly.